impl/cluster/pom.xml

```diff
@@ -15,6 +15,7 @@
   <url>http://www.pentaho.com</url>
   <properties>
     <publish-sonar-phase>site</publish-sonar-phase>
+    <dependency.org.powermock.version>1.7.3</dependency.org.powermock.version>
```

Comment:
Please move this to the top level POM in this repository.
The correct name for this property should be `powermock.version`.
Usage of `dependency.` is deprecated.

Reply:
Done